modules/openapi-generator/src/main/resources/python/asyncio/rest.mustache

```diff
@@ -68,6 +71,20 @@ class RESTClientObject:
             trust_env=True
         )
 
+        self.retries = configuration.retries
+        if self.retries is not None:
+            self.retry_client = aiohttp_retry.RetryClient(
+                client_session=self.pool_manager,
+                retry_options=aiohttp_retry.ExponentialRetry(
+                    attempts=self.retries,
+                    factor=0.0,
+                    start_timeout=0.0,
+                    max_timeout=120.0
+                )
+            )
+        else:
+            self.retry_client = None
+
     async def close(self):
         await self.pool_manager.close()
 
```

Comment:
From the aiohttp_retry documentation, it looks like RetryClient needs to be closed
https://github.com/inyutin/aiohttp_retry#examples-of-usage

Reply:
Closing the retry client just closes the session which is taken care of by the RESTClientObject.close() here.

Reply:
It probably doesn't hurt to add the close() call and not to assume too much what RetryClient is doing though.
```suggestion
        if self.retry_client is not None:
            await self.retry_client.close()

```